python/gen_requirements.py

```diff
@@ -64,6 +64,7 @@
         (
             "Base requirements needed to install tvm",
             [
+                "Pygments",
```

Comment:
You need to add a version constraint for Pygments in the `CONSTRAINTS` array lower in this file.